yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -33,8 +33,12 @@ type TargetDB interface {
 	ImportBatch(batch Batch, args *ImportBatchArgs) (int64, error)
 	IfRequiredQuoteColumnNames(tableName string, columns []string) ([]string, error)
 	ExecuteBatch(batch []*Event) error
+	GetDebeziumValueConverterSuite(isStreamingMode bool) map[string]ConverterFn
 }
 
+//value converter Function type 
+type ConverterFn func(v string) (string, error)
```

Comment:
`ConverterFn` should take one more boolean argument `quoteIfRequired`. If the flag is true, the return value should be such that it can be directly passed in a SQL statement (INSERT, UPDATE, etc.). If the flag is false, it should return values suitable to be written in a batch file.